translations/mgcp.js

```diff
@@ -2036,6 +2036,23 @@ mgcp = {
 
             mgcp.lookup = translate.createLookup(mgcp.rules.one2one);
         }
+
+        if (mgcp.txtLookup == undefined)
+        {
+            mgcp.txtLookup = {};
+            // Add the MGCPv3.0 specific attributes to the v4.0/common attribute table
+            for (var i in mgcp.rules.txtBiased) mgcp.txtLookup[i] = mgcp.rules.txtBiased[i];
+            for (var i in mgcp.rules.txtBiasedV3) mgcp.txtLookup[i] = mgcp.rules.txtBiasedV3[i];
```

Comment:
Splitting TRD3 & TRD4 values into their own lists.
The lists are added when going MGCP -> OSM
Only TRD4 is used when going OSM-> MGCP